src/SNPN/Common/NotificationService.cs

```diff
@@ -76,7 +79,12 @@ private async Task ProcessNotificationQueue()
 								case NotificationType.Badge:
 								case NotificationType.Tile:
 								case NotificationType.Toast:
-									result = await _networkService.SendNotification(notification, token);
+									if(notification.Uri.StartsWith("fcm://")) {
```

Comment:
Probably a better way of handling this, but oh well :|

Reply:
Without adding a registration type to the DB, yeah this is probably the best way. I'm ok with it since migrating existing users and changing the schema would be a pain.